module/applications/actor/group-sheet.mjs

```diff
@@ -353,6 +360,21 @@ export default class GroupActorSheet extends ActorSheet {
 
   /* -------------------------------------------- */
 
+  /**
+   * Change the quantity or limited uses of an Owned Item within the actor.
+   * @param {Event} event        The triggering click event.
+   * @returns {Promise<Item5e>}  Updated item.
+   * @private
+   */
+  async _onQuantityChange(event) {
+    const proto = game.system.applications.actor.ActorSheet5e.prototype;
```

Comment:
It's fine to call into the `ActorSheet5e` `prototype` for now, but we should call this method something different since it doesn't pertain only to quantity anymore. Something like `_onInlineItemChange` maybe. I'm open to suggestions.